src/ucp/core/ucp_listener.c

```diff
@@ -246,6 +247,38 @@ ucp_listen(ucp_listener_h listener, const ucp_listener_params_t *params)
     return status;
 }
 
+static void ucp_listener_vfs_show_ip(void *obj, ucs_string_buffer_t *strb)
+{
+    ucp_listener_h listener   = obj;
+    struct sockaddr *sockaddr = (struct sockaddr*)&listener->sockaddr;
+    char ip_str[UCS_SOCKADDR_STRING_LEN];
+
+    if (inet_ntop(sockaddr->sa_family, ucs_sockaddr_get_inet_addr(sockaddr),
```

Comment:
i'd suggest to add helper function fo sock.h/c to return the string of net part: ucs_sockaddr_get_ipstr(), and reuse it in ucs_sockaddr_str() function

Reply:
@yosefe please find the changes here #6678